src/clusterfuzz/_internal/fuzzing/fuzzer_selection.py

```diff
@@ -135,8 +135,23 @@ def get_fuzz_task_payload(platform=None):
   if not mappings:
     return None, None
 
+  logs.log(f'Mappings: {mappings}')
+  # The environment variable contains the list of jobs seperated by comma
+  # E.g: "libfuzzer_asan_android_host, afl_asan_android_host, ..."
```

Comment:
I know you `strip()` the jobs in `get_job_list()` but for the example it may make sense to not have spaces after the comma

Reply:
Done